Makefile

```diff
@@ -78,52 +97,59 @@ create_cluster:
 	# for virsh commands in .ssh/authorized_keys
 	set -e;                                               \
 	if ! grep -qf id_rsa.pub ~/.ssh/authorized_keys; then \
-	    (echo -n "command=\"$$PWD/vagrant-virsh\" ";      \
-	     cat id_rsa.pub) >> ~/.ssh/authorized_keys;       \
+	    echo "working directory: $$PWD";                  \
+	    if [ -e $(VAGRANT_VM3_LIBVIRT_DIR)/id ]; then     \
+	        (echo -n "command=\"$$PWD/vagrant-virsh\" ";  \
+	         cat id_rsa.pub) >> ~/.ssh/authorized_keys;   \
+	    else                                              \
+	         cat id_rsa.pub >> ~/.ssh/authorized_keys;    \
```

Comment:
Adding this key should be guarded off into a forced-command.  It would not be nice to add a key that has unlimited access to the host system.

Reply:
don't think we need it actually

Reply:
You probably will when you want to use the virtualbox fencing agent.